test/helpers/cilium.go

```diff
@@ -732,7 +732,7 @@ func (s *SSHMeta) DumpCiliumCommandOutput() {
 	// No need to create file for bugtool because it creates an archive of files
 	// for us.
 	res := s.ExecWithSudo(
-		fmt.Sprintf("%s -t %s", CiliumBugtool, filepath.Join(s.basePath, testPath)),
+		fmt.Sprintf("%s -t \"%s\"", CiliumBugtool, filepath.Join(s.basePath, testPath)),
```

Comment:
Maybe use `%q` instead.

Reply:
Thanks! Fixed and tested locally that it also works.